pkg/agent/route/route_windows.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package route
+
+import (
+	"fmt"
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/types"
+	"net"
+
+	"github.com/rakelkar/gonetsh/netroute"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/util"
+)
+
+type hRoute struct {
```

Comment:
what h stands for? host? how about using the full name since it's not too long and can map to HostRoute interface clearly?

Reply:
I will update.